.github/workflows/push-docker-production.yml

```
@@ -0,0 +1,46 @@
+name: push-docker-production
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build-push-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push latest
+        uses: docker/build-push-action@v3
+        with:
+          context: "{{defaultContext}}:rails"
+          push: true
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
+          tags: terrastories/terrastories:latest
```

I think we'll need to update this to specify the correct folder, otherwise this will use the base Dockerfile, which will not be correctly built for standalone use.

```suggestion
        with:
          context: "{{defaultContext}}:rails"
          file: "standalone.Dockerfile"
          push: true
          platforms: linux/amd64,linux/arm64,linux/arm/v7
          tags: terrastories/terrastories:latest
```